app/views/fileChecksProgress.scala.html

```diff
@@ -9,8 +9,14 @@
             @progressIndicator(Messages("checkingRecords.progress"))
             <h1 class="govuk-heading-xl">@Messages("checkingRecords.title")</h1>
             <p class="govuk-body">@Messages("checkingRecords.description")</p>
-            <p class="govuk-body">@Messages("checkingRecords.antivirus")</p>
-            <progress id="av-metadata-progress-bar" class="progress-display" value="@avMetadataPercentage" max="100"></progress>
+            <p>
+                <label for="av-metadata-progress-bar" class="govuk-body">@Messages("checkingRecords.antivirus")</label>
```

Comment:
Just to check - what's the advantage of putting this header into a `label` element? My understanding was that `label` was for form elements.

Reply:
From what I understood, the label tag enables it to be read out aloud by screen readers - https://www.w3schools.com/tags/tag_label.asp

W3schools and Mozilla both use it in conjunction with the progress tag 
- https://www.w3schools.com/tags/tag_progress.asp
- https://developer.mozilla.org/en-US/docs/Web/HTML/Element/progress

Reply:
I didn't realise that, thanks!